opm/simulators/wells/WellInterfaceGeneric.hpp

```diff
@@ -214,6 +214,14 @@ class WellInterfaceGeneric {
     bool wellUnderZeroRateTarget(const SummaryState& summary_state,
                                  const WellState& well_state) const;
 
+    std::tuple<bool,bool>
```

Comment:
Just out of curiosity: Why `tuple<>` instead of `pair<>` here?

Reply:
no reason, changed.